pkg/cephfs/volumemounter.go

```diff
@@ -54,12 +54,27 @@ func loadAvailableMounters() error {
 	// #nosec
 	kernelMounterProbe := exec.Command("mount.ceph")
 
-	if fuseMounterProbe.Run() == nil {
-		availableMounters = append(availableMounters, volumeMounterFuse)
+	if kernelMounterProbe.Run() == nil {
+		kernelVersion, _, err := execCommand("uname", "-r")
+		if err != nil {
+			return err
+		}
+		vers := strings.Split(string(kernelVersion), ".")
+		majorVers, err := strconv.Atoi(vers[0])
+		if err != nil {
+			return err
+		}
+		minorVers, err := strconv.Atoi(vers[1])
+		if err != nil {
+			return err
+		}
+		if majorVers >= 4 && minorVers >= 17 {
```

Comment:
else, log a warning that kernel client < 4.17 can't be used as a mounter as it doesn't support quotas?

Reply:
@poornimag please add a log message if kernel client is less than expected version